cat-launcher/src-tauri/src/variants/commands.rs

```diff
@@ -0,0 +1,38 @@
+use tauri::{command, State};
+use crate::variants::GameVariant;
+use crate::variants::get_game_variants_info::{self, GameVariantInfo};
+use crate::variants::repository::sqlite_game_variant_order_repository::SqliteGameVariantOrderRepository;
+use crate::variants::update_game_variant_order;
+use crate::settings::Settings;
+
+#[derive(thiserror::Error, Debug, serde::Serialize, strum::IntoStaticStr)]
+pub enum UpdateGameVariantOrderCommandError {
+    #[error("failed to update game variant order")]
+    Update(String),
+}
+
+#[derive(thiserror::Error, Debug, serde::Serialize, strum::IntoStaticStr)]
+pub enum GetGameVariantsInfoCommandError {
+    #[error("failed to get game variant order")]
+    Get(String),
```

Comment:
Use #from to convert from the correct error type to Get. Don't use String. See how other errors are defined.